src/Adapter/CustomerService/CommandHandler/ReplyToCustomerThreadHandler.php

```diff
@@ -138,7 +138,7 @@ private function sendReplyEmail(CustomerThread $customerThread, CustomerMessage
         $contact = new Contact((int) $customerThread->id_contact, (int) $customerThread->id_lang);
 
         if (Validate::isLoadedObject($contact)) {
-            $fromName = $contact->name;
+            $fromName = is_array($contact->name) ? reset($contact->name) : $contact->name;
```

Comment:
```suggestion
            $fromName = is_array($contact->name) ? $contact->name[0] : $contact->name;
```

Why not? Sorry I always fear `reset` because I had a bad experience in the past 😅

Reply:
Fixed